RecoLocalTracker/SiPixelRecHits/interface/PixelCPEBase.h

```diff
@@ -85,10 +85,10 @@ class PixelCPEBase : public PixelClusterParameterEstimator
    
    struct ClusterParam
    {
-      ClusterParam(const SiPixelCluster & cl) : theCluster(&cl), loc_trk_pred(0.0,0.0,0.0,0.0),
-      probabilityX_(0.0), probabilityY_(0.0), probabilityQ_(0.0), qBin_(0.0),
-      isOnEdge_(false), hasBadPixels_(false), spansTwoROCs_(false), hasFilledProb_(false) {}
```

Comment:
not everything here now have default values or is default constructed.
probs, qBin_, and spansTwoROCs_ are not initialized.
Is it quantifiably faster?
Better keep initialized.